File: python/examples/example_portcalls_imo.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.port_calls(imo=9175717, interval=10080))
except ApiErrorException as e:
    print(e)

File: php/examples/example_portcalls_imo.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $port_calls = $client->portCalls(10080, ['imo' => 9723708]);
    print_r($port_calls);
    print_r($client->getLastInfo());
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}